### PR TITLE
[REFACTOR]: Big Catado

### PR DESCRIPTION
**Pull request by rogefm**

### CODE REVIEW

File: webapp/src/webapp/webclient/aside/main.cljs [hunk 1]
```diff
@@ -22,110 +22,126 @@
...
+                                                        true
+                                                        false)}
+                  [connections-running-list/main {:schema-disabled? schema-disabled?
+                                                  :show-tree? show-tree?
+                                                  :run-connections-list-selected run-connections-list-selected}]]]))]])
          [:div
-          [:> ui/Disclosure {:defaultOpen true}
+          [:> ui/Disclosure {:defaultOpen is-connection-section-open?}
            (fn [params]
+             (println (.-open params))
```
Comment (mtmr0x) [inline webapp/src/webapp/webclient/aside/main.cljs]: 👀

File: webapp/src/webapp/webclient/log_area/main.cljs [hunk 2]
```diff
@@ -86,18 +102,13 @@
...
+          [tabs {:on-click (fn [_ value]
+                             (.setItem js/localStorage "webclient-selected-tab" value)
+                             (reset! selected-tab value))
+                 :tabs available-tabs
                  :selected-tab @selected-tab}]
           (case @selected-tab
             "AI" [terminal/main :ai ai-content]
             "Tabular" [data-grid-table/main results-heads results-body true tabular-loading?]
-            "Terminal" [terminal/main :terminal terminal-content]
+            "Logs" [terminal/main :terminal terminal-content]
```
Comment (mtmr0x) [inline webapp/src/webapp/webclient/log_area/main.cljs]: I'd add some refactoring to these names as well so they can reflect what they are. Also, we have a "terminal" like experience in the upcoming future, having things with their names all refactored on these changes will help that

File: webapp/src/webapp/webclient/panel.cljs [hunk 3]
```diff
@@ -63,20 +69,27 @@
       (reset! timer (js/setTimeout #(save-code-to-localstorage code-string) 1000))
       (reset! script code-string))))
 
-(defn- submit-task [e script selected-connections atom-exec-list-open? metadata]
-  (when (.-preventDefault e) (.preventDefault e))
+(defn- submit-task [e script selected-connections atom-exec-list-open? metadata script-response]
+  (println @script-response)
```
Comment (mtmr0x) [inline webapp/src/webapp/webclient/panel.cljs]: feature request to GitHub, let me add gifs from here without having to go to giphy.com

File: webapp/src/webapp/webclient/panel.cljs [hunk 4]
```diff
@@ -160,10 +174,7 @@
             review-plugin->connections (map #(:name %) (:connections (get-plugin-by-name "review")))
             current-connection last-connection-selected
             connection-name (:name current-connection)
-            connection-type (cond
-                              (not (cs/blank? (:subtype current-connection))) (:subtype current-connection)
-                              (not (cs/blank? (:icon_name current-connection))) (:icon_name current-connection)
-                              :else (:type current-connection))
+            connection-type (discorver-connection-type current-connection)
```
Comment (mtmr0x) [inline webapp/src/webapp/webclient/panel.cljs]: there's a typo here in the `discover` word, a rogue R

**Review by mtmr0x — CHANGES_REQUESTED** [review]

File: webapp/src/webapp/webclient/aside/main.cljs (hunk 1 above)
Comment (rogefm) [reply webapp/src/webapp/webclient/aside/main.cljs]: Sorry, it is still in progress. I removed all println
Thx to note.

**Pull request merged**